app/settings.py

```diff
@@ -126,7 +126,7 @@ def utcoffset_or_fail(date_value, key):
 EQ_SESSION_ID = "eq-session-id"
 
 EQ_LIST_ITEM_ID_LENGTH = 6
-MAX_NUMBER = 9999999999
+MAX_NUMBER = 999_999_999_999_999
```

Comment:
Just commenting this here as I'm not sure where it should be exactly:

The negative limit is -999999999999999, 15 digits instead of 14 as shown in screenshot:

<img width="841" alt="Screenshot 2023-04-20 at 16 40 06" src="https://user-images.githubusercontent.com/108286538/233417447-98917299-d4cd-4971-a724-83199f094ceb.png">

Reply:
Yeah that is a schema problem . I just need to adjust the minimum values in the `test_currency `schema to 14 now instead of 12

Reply:
It's not a ticket AC but it is in the tech details that the min should be negative 14 9s, which I think should be enforced in runner, not in the schema. Maybe someone could clarify this?

Reply:
Right now default the minimum to 0, on line 41. If u change that value to take the `MIN_NUMBER`. It will be used in schemas that may not want negative answers, like in stocks schemas since they don't speicfy the min number. Not sure if we can change this, any other devs have a input on this?
<img width="719" alt="image" src="https://user-images.githubusercontent.com/113098086/234227509-3ed3860d-77e6-4d04-b8bb-bc1d6207b7b2.png">

Reply:
No changes should be needed in runner. Validator currently checks schema min is greater than the min limit. However, it looks like it only checks static values and any reference values using answer sources are not checked, so we could update that. It is done in `NumberAnswerValidator.validate_value_in_limits`. There is a method on questionnaire schema to get a referred value from an answer source: `_get_numeric_value_for_value_source`